src/modules/form/components/DynamicField.tsx

```diff
@@ -364,11 +374,12 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
             placeholder={placeholder}
             textInputProps={{
               horizontal,
-              sx: {
-                maxWidth: MAX_INPUT_AND_LABEL_WIDTH,
-                '.MuiInputBase-root': { maxWidth: MAX_INPUT_WIDTH },
-              },
+              InputProps: { sx: { width } },
```

Comment:
Touching this code made me wonder, what was behind the original desire to have 600px wide labels but 500px wide inputs in general -- so that these Select components are the exception to that rule? Maybe something to check in with Ash about